src/js/subtraction/components/Detail/EditSubtraction.tsx

```diff
@@ -24,7 +24,7 @@ type EditSubtractionProps = {
 };
 
 /**
- * A modal for editing the subtraction
+ * Displays a modal for editing a subtraction
```

Comment:
Nit: The current goal for component docstrings is to describe what they are rather than being action focused like a normal function. 

```suggestion
 * Dialog for editing an existing subtraction
```
or something like that.